common/src/main/resources/data/sql/schema/mysql.sql

```diff
@@ -85,4 +85,6 @@ CREATE TABLE journey_tunnels (
     )
 );
 
+-- I don't include the DB versions table here since it's automatically created.
+
```

Comment:
The purpose of having these schema files is to dictate how the *current* system looks like.

It is the job of database migrations to update any outdated schemas to the current system. Otherwise, if a system is starting from scratch, then this schema file should be used.

So, here we should add the version tables, and additionally, create the table manually in the code when we identify that the system is on database `V1` and needs to update to `V2`. For most migrations, we can write it as raw SQL, which is clean and can be written up in a resource `.sql` file. Let's do that.

Reply:
Using SQL files to migrate is a great idea, no idea why I didn't think of that.